### ISSUE TITLE
Live queries do not update when deleting rows

### ISSUE THREAD

**Issue by ni-ra** [graphile/postgraphile#1033]
I'm submitting a ... 

* [X] bug report
* [ ] feature request
* [ ] question 

PostGraphile version:

4.4.0-beta.8  (postgraphile and subscriptions-lds)

Steps to reproduce:

postgraphile --append-plugins @graphile/subscriptions-lds --live -c \<connection-string\>  

Current behavior:  
Live queries do not update when something is deleted. Further investigation showed that the lds server does not send updates upon the deletion of rows.

Expected behavior:
Live queries update after deleting a row.

**Comment by benjie** [comment]
Make sure you’re using @graphile/subscriptions-lds@next

**Comment by benjie** [comment]
I've updated the tag so `@latest` now points to 4.4.0-beta.8 instead of 4.4.0-alpha.0 (which is what I suspect you have installed); so a regular update should fix it now. Until 4.4.0 is officially released I suggest you stick to the `@next` versions, e.g.:


```
yarn add \
  postgraphile@next \
  graphile-utils@next \
  @graphile/subscriptions-lds@next \
  @graphile/pg-pubsub@next
```

**Comment by ni-ra** [comment]
Thank you for the fast response. 
I just verified the version again (and reinstalled all those packages @next).
Everything seems to be working, besides updates on deletion.

**Comment by benjie** [comment]
I'm going to take a look at this tomorrow hopefully.

**Comment by frankdugan3** [comment]
Here's a [full repro (see README)](https://github.com/frankdugan3/postgraphile-lds-add-delete-bug).

**Comment by benjie** [comment]
Fixed in #1037 (hopefully); released in `postgraphile@4.4.0-beta.9` (i.e. `postgraphile@next`).

```
yarn add postgraphile@next @graphile/subscriptions-lds@next
```
